front/app/modules/commercial/representativeness/admin/utils/bins.ts

```diff
@@ -9,7 +9,7 @@ export const getBinId = (
 ) =>
   upperBound === null
     ? `${lowerBound}+`
-    : `${lowerBound}-${upperBound - (isLastBin ? 0 : 1)}`;
+    : `${lowerBound} - ${upperBound - (isLastBin ? 0 : 1)}`;
```

Comment:
I added a space here so that we could reuse this `binId` for the tick labels in the graph